# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1437782298 14400
#      Fri Jul 24 19:58:18 2015 -0400
# Node ID 39721c09691bc19cd045272fcaf94a30eacb8a05
# Parent  41d19a6ef55a46b8005fef83d21cee8e8e7bb794
build: Delete more generated files on maintainer-clean

* Makefile.am (MAINTAINERCLEANFILES): Include $(BUILT_DISTFILES) in the list.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -158,16 +158,19 @@ OCTAVE_INTERPRETER_TARGETS += run-octave
 CLEANFILES += \
   $(BUILT_SOURCES)
 
 DISTCLEANFILES += \
   .gdbinit \
   ChangeLog \
   $(DIRSTAMP_FILES)
 
+MAINTAINERCLEANFILES += \
+  $(BUILT_DISTFILES)
+
 CONFIG_FILES = @ac_config_headers@ @ac_config_files@
 
 octinclude_HEADERS += oct-conf-post.h
 nodist_octinclude_HEADERS += config.h
 
 all-local: $(ALL_LOCAL_TARGETS) $(noinst_SCRIPTS) $(DIST_INFO_FILES) .gdbinit $(DOC_TARGETS)
 	@echo ""
 	@echo "Octave successfully built.  Now choose from the following:"
